`make operator-developer-e2e` fails when container with name "local-registry" is already exists

```
docker run -d -p 5001:5000 --restart=always --name local-registry registry:2
Error: creating container storage: the container name "local-registry" is already in use by 7a2822e1ac9473f75a0f1e1e42ae0704b2d3123b18c49181a089e4624feea350. You have to remove that container to be able to reuse that name: that name is already in use
make: *** [Makefile:154: deploy-local-registry] Error 125
````

--- a/Makefile
+++ b/Makefile
@@ -129,6 +129,7 @@ kind-load: $(KIND) ## Loads the currently constructed image onto the cluster
 
 .PHONY: kind-cluster
 kind-cluster: $(KIND) ## Standup a kind cluster
+	-$(KIND) delete cluster --name ${KIND_CLUSTER_NAME}
 	$(KIND) create cluster --name ${KIND_CLUSTER_NAME}
 	$(KIND) export kubeconfig --name ${KIND_CLUSTER_NAME}
 

